app/views/layouts/support.html.erb

```diff
@@ -6,17 +6,16 @@
     ) %>
   <% end %>
 
-<!-- Uncomment this when more items are needed on the navigation bar.
-  <%# <%= content_for(:navigation_bar) do %> %>
-    <%# <%= render NavigationBar.new( %>
-      <%# items: [ %>
-        <%# { name: "Providers", url: support_providers_path, current: true }, %>
-      <%# ], %>
-      <%# current_path: request.path, %>
-      <%# current_user: @current_user, %>
-    <%# ) %> %>
-  <%# <% end %> %>
--->
+  <%= content_for(:navigation_bar) do %>
+    <%= render NavigationBar.new(
+      items: [
+        { name: "Providers", url: support_recruitment_cycle_providers_path(params[:recruitment_cycle_year] || Settings.current_recruitment_cycle_year) },
+        { name: "Users", url: support_recruitment_cycle_users_path(params[:recruitment_cycle_year] || Settings.current_recruitment_cycle_year) }
```

Comment:
Our proto looks like this, which I believe was ahead of production, although they would have slightly diverged now since we did some work on support:

<img width="1164" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/cd582308-647d-415d-95b8-9c6ce78681b8">

Reply:
Interesting around the prototype! on the users though, one is all users right, and the other is just users relating to that provider?

Reply:
Yes, it's not confusing for us, but I think it might be for others.

How about we change the one in the primary nav to something else, maybe `All users`? what do you think? Just a way to clearly differentiate between the two.